packages/wallet/api/src/lib-dehydrate.js

```diff
@@ -319,11 +328,16 @@ export const makeDehydrator = (initialUnnamedCount = 0) => {
   const convertValToName = val => {
     for (let i = 0; i < searchOrder.length; i += 1) {
       const kind = searchOrder[i];
-      const { valToPetname } = petnameKindToMapping.get(kind);
+      const { valToPetname, valToBoardId } = petnameKindToMapping.get(kind);
       if (valToPetname.has(val)) {
+        let boardId;
+        if (valToBoardId.has(val)) {
+          boardId = valToBoardId.get(val);
+        }
         return harden({
           kind,
           petname: valToPetname.get(val),
+          boardId,
```

Comment:
any particular reason to include the `boardId` property when it's undefined?

Reply:
Nope, removed now.